### PR TITLE
Refactor documentation - Do not add annotation configuration if you are using Symfony Flex

### PR DESCRIPTION
**Pull request by idetox**
Do not add annotation routing configuration if your project uses Symfony Flex since the recipe already added the annotations.yaml file with the config inside.

### CODE REVIEW

File: best_practices/controllers.rst [hunk 1]
```diff
@@ -47,14 +47,14 @@ this suffix is neither required nor recommended, so you can safely remove it.
 Routing Configuration
 ---------------------
 
-To load routes defined as annotations in your controllers, add the following
-configuration to the main routing configuration file:
+To load routes defined as annotations in your controllers and if you are not using Symfony Flex, add the following
+configuration to the annotation routing configuration file:
```
Comment (weaverryan) [inline best_practices/controllers.rst]: Great find with this. I think we can go further and fully assume Flex and reword this section to read something more like this:

> To load routes defined as annotations in your controllers, run “composer require doctrine/annotations”. Thanks to the Flex recipe, a config/routes/annotations.yaml file will be created:

... show the file

... then continue on like normal with “This Configuration...”

**Comment by weaverryan** [comment]
Status: needs work

File: best_practices/controllers.rst (hunk 1 above)
Comment (idetox) [reply best_practices/controllers.rst]: Should we spread this modification to other branches ?
If so, i guess i should duplicate the PR once you validate it ? Or will you be able to do it automatically ?

File: best_practices/controllers.rst [hunk 2]
```diff
@@ -47,8 +47,13 @@ this suffix is neither required nor recommended, so you can safely remove it.
 Routing Configuration
 ---------------------
 
-To load routes defined as annotations in your controllers and if you are not using Symfony Flex, add the following
-configuration to the annotation routing configuration file:
+To load routes defined as annotations in your controllers, run the following command :
```
Comment (OskarStark) [inline best_practices/controllers.rst]: Please remove the space before the colon

**Comment by idetox** [comment]
@weaverryan did you have time to review ?
thx !

**Comment by javiereguiluz** [comment]
@idetox thanks for helping us improve this! I did some minor rewords while merging to mach Ryan's suggestions.

**Pull request merged**